e2e/controls.py

```diff
@@ -2332,3 +2332,186 @@ def expect_n_row(
             n,
             timeout=timeout,
         )
+
+
+class ValueBox(
+    _WidthLocM,
+    _InputWithContainer,
+):
+    # title: TagChild,
+    # value: TagChild,
+    # *args: TagChild | TagAttrs,
+    # showcase: TagChild = None,
+    # showcase_layout: ((TagChild, Tag) -> CardItem) | None = None,
+    # full_screen: bool = False,
+    # theme_color: str | None = "primary",
+    # height: CssUnit | None = None,
+    # max_height: CssUnit | None = None,
+    # fill: bool = True,
+    # class_: str | None = None,
+    # **kwargs: TagAttrValue
+    def __init__(self, page: Page, id: str) -> None:
+        super().__init__(
+            page,
+            id=id,
+            loc_container=f"div#{id}.bslib-value-box",
+            loc="> div > .value-box-grid",
+        )
+        value_box_grid = self.loc
+        self.loc = value_box_grid.locator(
+            "> div > .value-box-area > :not(:first-child)"
+        )
+        self.loc_showcase = value_box_grid.locator("> div > .value-box-showcase")
+        self.loc_title = value_box_grid.locator(
+            "> div > .value-box-area > :first-child"
+        )
+        self._loc_fullscreen = self.loc_container.locator("> .bslib-full-screen-enter")
```

Comment:
Just a heads-up, we recently changed the selector for this on the R side. So when py-shiny updates it's card css, this selector will need to change (cc @schloerke)

https://github.com/rstudio/bslib/pull/669/files